CHANGELOG.md

```diff
@@ -3,6 +3,117 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.11
+
+_Dec 8, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add dragging support for `DateRangePicker` (#6763) @LukasTy
```

Comment:
The range support is not available on the `DateRangePIcker` component, only on `NextDateRangePicker` and `DateRangeCalendar`.

Maybe rephrase to something like

```suggestion
- 🚀 Add dragging support for the new Date Range Picker (`NextDateRangePicker`) (#6763) @LukasTy
```

To avoid any confusion